rest-api/plugin.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?eclipse version="3.4"?>
+<plugin>
+   <extension point="org.eclipse.ui.startup">
+       <startup
+           class="com.db.capella.Main">
```

Comment:
The `com.db.*` namespace belongs to Deutsche Bank (<https://db.com> redirects to them). Please change this to either `de.db.*` or `com.deutschebahn.*`.

Reply:
good point. done.